### ISSUE TITLE
How to use this? There is no script.

### ISSUE THREAD

**Issue by deathxxx123** [iAmSomeone2/asus_bios_renamer#1]
I want to do some rename bios files from linux, but don't see how i can install this.

**Comment by carlosbanales** [comment]
Im not sure if this is how your suppose to but this worked for me.

1. Confirm Rust is installed on your machine
2. Navigate to `bios_renamer_for_asus/src`
3. Run command `rustc main.rs`
4. Use the new executable file created and run `./main path/to/file`

**Comment by iAmSomeone2** [comment]
Sorry for not jumping in here earlier. Work/life has been keeping me away from my hobby projects. 

When I wrote this, I was looking for an excuse to learn Rust, and making this tool was a good, simple project to get a start with that. I definitely didn't expect it to get any attention. Hence why there's no build/install script or prebuilt packages.

Given that there seems to be some interest, I'll try to carve out a little time to make some packages. Until I get around to that, follow the instructions that @carlosbanales posted.

tl;dr
@carlosbanales is correct. Use his instructions until I can get around to making some packages.
